### PR TITLE
docker: fix ulimits for eos container

### PR DESCRIPTION
**Pull request by labkode**

### CODE REVIEW

File: docker/Dockerfile.revad-ceph [hunk 1]
```diff
@@ -20,7 +20,6 @@ FROM quay.io/ceph/ceph:v18
 
 # replace repo url with one that allows downloading the repo metadata
 # if http://download.ceph.com/rpm-reef/el8/x86_64/repodata/repomd.xml works again this can be dropped
```
Comment (glpatcern) [inline docker/Dockerfile.revad-ceph:22]:
```suggestion
```

File: tests/docker/eos-storage/Dockerfile [hunk 2]
```diff
@@ -3,8 +3,13 @@ FROM gitlab-registry.cern.ch/dss/eos/eos-ci:5.1.25
 COPY scripts/eos-run.sh /mnt/scripts/eos-run.sh
 COPY sssd/sssd.conf /etc/sssd/sssd.conf
 
-RUN ulimit -n 1024000 && yum install -y sssd sssd-client
+# RUN ulimit -n 1024000 && yum install -y sssd sssd-client
+# needs to be run from docker like this:
+# docker run myeoscontainer --ulimit nofiles:1024000:1024000
+# or in the ulimits directive for docker compose
+
```
Comment (glpatcern) [inline tests/docker/eos-storage/Dockerfile:10]:
```suggestion
```

Comment (glpatcern) [reply tests/docker/eos-storage/Dockerfile:10]: (do we need that?)

**Review by glpatcern — COMMENTED** [review]
Minimal cleanups...

File: tests/docker/eos-storage/Dockerfile (hunk 2 above)
Comment (labkode) [reply tests/docker/eos-storage/Dockerfile]: I've kept the documentation as is needed to debug issues.

File: docker/Dockerfile.revad-ceph (hunk 1 above)
Comment (labkode) [reply docker/Dockerfile.revad-ceph]: Fixed

**Review by glpatcern — APPROVED** [review]

**Pull request merged**